tutorials/raw/plot_annotating_raw.py

```diff
@@ -0,0 +1,225 @@
+# (:func:`~mne.preprocessing.find_ecg_events`,
+# :func:`~mne.preprocessing.find_eog_events`), and creation of epoched data
+# from continuous data (:class:`mne.Epochs`).
+#
+# .. TODO: change "projection" in prev. para to this when tut is ready:
+#    :ref:`projection <ssp-tutorial>`
+#
+#
+# Operations on :class:`~mne.Annotations` objects
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# :class:`~mne.Annotations` objects can be combined by simply adding them with
+# the ``+`` operator, as long as they share the same ``orig_time``:
+
+new_annot = mne.Annotations(onset=3.75, duration=0.75, description='AAA')
+raw.set_annotations(my_annot + new_annot)
+raw.plot(start=2, duration=6)
+
+###############################################################################
+# Notice that it is possible to create overlapping annotations, even when they
+# share the same description. This is *not* possible when annotating
+# interactively; click-and-dragging to create a new annotation that overlaps
+# with an existing annotation with the same description will cause the old and
+# new annotations to be merged.
+#
+# Individual annotations can be accessed by indexing an
+# :class:`~mne.Annotations` object, and subsets of the annotations can be
+# achieved by either slicing or indexing with a list, tuple, or array of
+# indices:
+
+print(raw.annotations[0])       # just the first annotation
+print(raw.annotations[:2])      # the first two annotations
+print(raw.annotations[(3, 2)])  # the fourth and third annotations
+
+###############################################################################
+# You can also iterate over the annotations within an :class:`~mne.Annotations`
+# object:
+
+for ann in raw.annotations:
+    descr = ann['description']
+    start = ann['onset']
+    end = ann['onset'] + ann['duration']
+    print(f"'{descr}' goes from {start} to {end}")
+
+###############################################################################
+# Note that iterating, indexing and slicing :class:`~mne.Annotations` all
+# return a copy, so changes to an indexed, sliced, or iterated element will not
+# modify the original :class:`~mne.Annotations` object.
+
+# later_annot WILL be changed, because we're modifying the first element of
+# later_annot.onset directly:
+later_annot.onset[0] = 99
+
+# later_annot WILL NOT be changed, because later_annot[0] returns a copy
+# before the 'onset' field is changed:
+later_annot[0]['onset'] = 77
+
+print(later_annot[0]['onset'])
+
+###############################################################################
+# :class:`~mne.Annotations` objects have a :meth:`~mne.Annotations.save` method
+# which can write :file:`.fif`, :file:`.csv`, and :file:`.txt` formats (the
+# format to write is inferred from the file extension in the filename you
+# provide). There is a corresponding :func:`~mne.read_annotations` function to
+# load them from disk:
+
+raw.annotations.save('saved-annotations.csv')
```

Comment:
use a temp directory like in the tests.

Reply:
using python tempdir can be scary for new comers and it can divert them from the important line of code. I think it's ok to write a tiny file somewhere on your disk

Reply:
then we should write it in '~/mne_data/tmp/', not in the code tree. We have functions to retrieve '$MNE_DATA'